docs/app.rst

```diff
@@ -0,0 +1,34 @@
+:mod:`gidgethub.app` --- Support for GitHub App
+===============================================
+
+.. module:: gidgethub.app
+
+.. versionadded:: 4.1.0
+
+This module is to help provide support for `GitHub App <https://developer.github.com/v3/apps/>`_.
+
+Example on how you would obtain the access token for authenticating as a GitHub App installation::
+
+    private_key = """-----BEGIN RSA PRIVATE KEY-----
+    zBgqFIin/uQEb0he006F9pNC6Kga0AMY5b0cCdZ4ge9qyFro2eVA
+    ...
+    -----END RSA PRIVATE KEY-----
+    """
+
+    access_token_response = await util.get_installation_access_token(installation_id=123, app_id=456, private_key=private_key)
```

Comment:
This isn't used anywhere. Is this the `access_token` variable referenced in the next line?

Reply:
yes, thanks for catching that. I've edited the next line.